src/quo2/components/loaders/skeleton/component_spec.cljs

```diff
@@ -0,0 +1,13 @@
+(ns quo2.components.loaders.skeleton.component-spec
```

Comment:
the namespace should just be identical to figma. i.e in this case `skeleton-list`, this helps avoid confusion as we know exactly what is in there. if we think there's a better name that can be used in the figma files then we should ask the designers to adjust it. namely @pedro-et & @mariocnovoa  👍

Reply:
Just pushed that